### ISSUE TITLE
New stance idea

### ISSUE THREAD

**Issue by IanFromArx** [Arx-Game/arxcode#90]
I was talking with a friend about combat in a different dice system than this one (Pathfinder), and it occurred to me that it might be useful to add to the stances 'Total Defense' and 'All Out Attack' to the system here.

In total defense, you wouldn't attack at all, but would have an absurd defense bonus.  This would be silly to use in sparring one on one (although I guess you could if you were waiting for fatigue to slow the other guy down, so maybe put a fatigue penalty on using it if you didn't want that to happen) but in a group combat, it would be good for situations where someone doesn't realize what they're getting into, winds up in over their head, and is trying to just hang on while their more powerful friends take out the bad guys.  Right now, the only solution for that is for one of said friends to guard you, but that's not always feasible.  It might lead to fewer deaths because a player just didn't understand the power level they were dealing with, and might occasionally allow non-combat characters like healers and scholars to come along on missions without worrying that they're going to get splattered.

All Out Attack would be the opposite -- a large initiative bonus and obscene combat bonuses, but you have no defense at all, so you're relying entirely on your hit points and mitigation and/or looking to go out with a bang.  This is for when someone absolutely HAS to hit the dragon in the eye with the magic sword because that's its only weakness (pair this with +attack/accuracy for exceptionally hard to hit targets) or when damn the consequences, that giant demon with ton of health has to go down (pair with +attack/critical for oodles of damage).

If someone could actually go berserk (or rage or frenzy -- I've played too much WoD), they might even be given access to a similar stance to all out attack, but without as many defense penalties, a small chance of attacking their friends, and the inability to switch stances until the end of the fight once berserk was chosen.

Anyway, it was just a thought.  I thought these two stances might fill a few holes in the coded system in terms of what people might want to do in a fight, and might give combats some more dramatic beats.

**Comment by IanFromArx** [comment]
Another use of All Out Attack, storywise?  (Because this has caught my imagination now.)  

"Hello.  My name is Inigo Montoya.  You killed my father.  Prepare to die."

Ie, 'I don't care what happens to me, but that six fingered monster who killed my father is going to pay.'

**Comment by TehomCD** [comment]
Related to #88 in that I think this all falls under the umbrella of combat revamp/extensions. I probably won't be adding anything until we can look at it in detail.

**Comment by Sritterbush** [comment]
As I recall you fatigue faster from being attacked than attacking, though I havn't looked into the system closely recently. There is already a pass option in combat to go the round without taking a swing.

Mechanically, we have the defend mechanic where person B can defend person A in combat. When used tactically, person A will probably use archery or boost their attack rolls on the belief person B will take the majority of the hits for them, and person B plays defensively. Against NPCs there is a metric in the system that reduces the effectiveness of defending others, making that tactic less reliable, but versus players it is incredibly potent.

Imagining that with someone about to do near-sure to hit attack/criticals while being defended seems like a rather big balance consideration.

**Comment by earthsend** [comment]
No, aggressive and reckless both increase your fatigue rolls' difficulty much faster. Defensive and guarded only start to increase your fatigue difficulty after 2+ combatants target you.
